packages/did/src/DidResolver/DidResolver.ts

```diff
@@ -219,12 +220,16 @@ export async function resolve(
 ): Promise<
   DidResolvedDetails | ResolvedDidKey | ResolvedDidServiceEndpoint | null
 > {
-  const { fragment } = parseDidUri(didUri)
+  const { fragment, did } = parseDidUri(didUri)
 
   if (fragment) {
-    return resolveKey(didUri) || resolveServiceEndpoint(didUri) || null
+    return (
+      resolveKey(didUri as DidResourceUri) ||
```

Comment:
Would this cause somehow issues if `didUri` is not a `DidResourceUri`? Like throwing or something?

Reply:
no, none of this survives compiling to js - in other words, there is no runtime checking whatsoever